services/ui-src/src/components/accordions/TemplateCardAccordion.test.tsx

```diff
@@ -5,23 +5,11 @@ import { RouterWrappedComponent } from "utils/testing/setupJest";
 // components
 import { TemplateCardAccordion } from "components";
 // data
-import templateCardsVerbiage from "../../data/templateCards.json";
+import data from "../../data/home-view.json";
 
-/*
- * From Chakra UI Accordion test file
- * https://bit.ly/3MFtwXq
- */
-jest.mock("@chakra-ui/transition", () => ({
-  ...jest.requireActual("@chakra-ui/transition"),
-  Collapse: jest.fn(({ in: inProp, children }: any) => (
-    <div hidden={!inProp}>{children}</div>
-  )),
-}));
```

Comment:
Moved this to `jestSetup` so the mock is available to any component that needs it.

Reply:
good call!